common/graphics/texture.cc

```diff
@@ -28,16 +38,36 @@ void TextureRegistry::UnregisterTexture(int64_t id) {
   mapping_.erase(found);
 }
 
+void TextureRegistry::UnregisterContextListener(uintptr_t id) {
+  images_.erase(id);
+}
+
 void TextureRegistry::OnGrContextCreated() {
   for (auto& it : mapping_) {
     it.second->OnGrContextCreated();
   }
+
+  for (const auto& [id, weak_image] : images_) {
+    if (auto image = weak_image.lock()) {
+      image->OnGrContextCreated();
```

Comment:
Would it make sense to do this lazily for each image instead of doing it for all images when the context becomes available?

Reply:
I think if we did it lazily it would be in the image itself rather than here - IOW the image object would have to track state and re-create the image on demand.

I'm inclined to think that it'd be better to just take time resuming than to potentially jank during an animation to re-create an image. I'm not really sure which will be better - if an application has a lot of these images it could significantly slow down resume, but there will be a number of problems with using that many images anyway e.g. memory constraints on the device.